analysis/deep2_r23_o32/archives/binning/binning_and_graphing_mastergrid.py

```diff
@@ -9,22 +9,24 @@
 def n_times_binned(fitspath, pdf_pages, outfile, n_split, individual_ID, R23, O32,
                    SNR3, data3, galinbin):
     """
-    R23 and O32 are going to be log values
-    One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
-    increase the number of galaxies as R23 increases
+    Purpose:
```

Comment:
Purpose
----------

Reply:
I created #64 for this issue. Will address outside this PR.